cmd/weaviate/main.go

```diff
@@ -110,8 +110,18 @@ func main() {
 		reflection.Register(grpcServer)
 		protocol.RegisterWeaviateServer(grpcServer, grpcQuerier)
 
+		enterrors.GoWrapper(func() {
+			metadataSubscription := query.NewMetadataSubscription(
+				a,
+				opts.Query.MetadataGRPCAddress,
+				log)
+			if err = metadataSubscription.Start(); err != nil {
```

Comment:
Q: I don't see it's stopping anywhere?

Plus I think may be it's worth accepting `ctx` and respect the cancellation, timeout, etc. given it's a blocking call?

Reply:
Thanks! I'll add the ctx in a follow up PR